docs/docs-content/release-notes/known-issues.md

```diff
@@ -14,18 +14,19 @@ to review and stay informed about the status of known issues in Palette. As issu
+| Description                                                                                                                                                                                                                                                                                                                                                 | Workaround                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Publish Date   | Product Component |
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------- | ----------------- |
+| Deploying a self-hosted Palette or VerteX, to a vSphere environment fails if vCenter has standalone hosts directly under a Datacenter. Persistent Volume (PV) provisioning fails due to an upstream issue with the vSphere Container Storage Interface (CSI) for all versions before v3.2.0. Palette and VerteX use the vSphere CSI version 3.12 internally | If you encounter the following error message: `'ProvisioningFailed failed to provision volume with StorageClass "spectro-storage-class". Error: failed to fetch hosts from entity ComputeResource:domain-xyz` then use the following workaround. Remove standalone hosts directly under the Datacenter from vCenter and allow the volume provisioning to complete. After the volume is provisioned, you can add the standalone hosts back. You can also use a service account that does not have access to the standalone hosts as the user that deployed Palette. | May 21, 2024   | Self-Hosted       |
```

Comment:
The other general comment is, this is also possible for other use case where a sphere cluster is deployed with the affected vsphere-csi storage driver. VOlume provisioning of the custom workloads will also experience the same issue.